src/pages/documentation/getting-started/configure-datastreams.md

```diff
@@ -1,40 +0,0 @@
-# Configure datastreams
```

Comment:
Also update gatsby-config for the deleted files

Reply:
Getting started does not appear in the Gatsby config